src/main/java/spaceinvaders/group_22/unit/SpaceShip.java

```diff
@@ -7,7 +7,7 @@
  * @author Dorian
  */
 @SuppressWarnings("checkstyle:magicnumber") 
-public class SpaceShip extends Unit implements MovableUnit {
+public class SpaceShip extends Unit implements MovableUnit, ShootableUnit {
```

Comment:
Might ShootingUnit not be a better name. Shootable made me think it was a bullet or some other object that "can be shot".